File: TransCoder_tokenized_test_set_functions/java/FREQUENCY_ELEMENT_UNSORTED_ARRAY_1.java
static void countFreq ( int a [ ], int n ) {
  int hm [ ] = new int [ n ];
  for ( int i = 0;
  i < n;
  i ++ ) hm [ a [ i ] ] ++;
  int cumul = 0;
  for ( int i = 0;
  i < n;
  i ++ ) {
    cumul += hm [ a [ i ] ];
    if ( hm [ a [ i ] ] != 0 ) {
      System . out . println ( a [ i ] + "->" + cumul );
    }
    hm [ a [ i ] ] = 0;
  }
}

File: TransCoder_tokenized_test_set_functions/python/FREQUENCY_ELEMENT_UNSORTED_ARRAY_1.py
def countFreq ( a, n ) :
    hm = dict ( )
    for i in range ( n ) :
        hm [ a [ i ] ] = hm . get ( a [ i ], 0 ) + 1
    cumul = 0
    for i in range ( n ) :
        cumul += hm [ a [ i ] ]
        if ( hm [ a [ i ] ] > 0 ) :
            print ( a [ i ], "->", cumul )
        hm [ a [ i ] ] = 0
    
